app/src/main/java/org/schulcloud/mobile/data/model/requestBodies/AddHomeworkRequest.java

```diff
@@ -0,0 +1,32 @@
+package org.schulcloud.mobile.data.model.requestBodies;
+
+import org.schulcloud.mobile.data.model.Course;
+
+public class AddHomeworkRequest
+{
+    String schoolId;
+    String teacherId;
+    String name;
+    Course courseId;
+    Boolean isPrivate;
+    String description;
+    String availableDate;
+    String dueDate;
+    Boolean publicSubmissions;
+
+    public AddHomeworkRequest(String schoolId, String teacherId, String name,
```

Comment:
This constructor has a lot of parameters. Maybe you solve this more elegant with the builder pattern. You can find a lot of example online.